apps/OpenSpace/ext/launcher/src/filesystemaccess.cpp

```diff
@@ -25,10 +25,18 @@
 #include "filesystemaccess.h"
 
 FileSystemAccess::FileSystemAccess(std::string fileExtension,
-                                   std::vector<std::string> approvedPaths,
                                    bool hideFileExtensions, bool useCheckboxes)
     : _fileExtension(std::move(fileExtension))
+    , _hideFileExtensions(hideFileExtensions)
+    , _useCheckboxes(useCheckboxes)
+{}
+
+FileSystemAccess::FileSystemAccess(std::string fileExtension,
```

Comment:
Are we using this constructor anywhere?  If not, I'd say to remove it and then also remove the `_useOnlyApprovedPaths` entirely

Reply:
I removed all of the "approved paths" filtering in 3572de5.